*All Models Testcases

Adeel-Tahir opened the pull request:
*Categories and Items Controller Testcases

Adeel-Tahir committed:
*All Models Testcases
*Categories and Items Controller Testcases

Adeel-Tahir committed:
*fix changes of test-cases

ziaulrehman40 commented on spec/concerns/validatable_spec.rb:
```diff
@@ -0,0 +1,12 @@
+
+require 'rails_helper'
+
+shared_examples_for 'validatable' do
```
not used

ziaulrehman40 commented on spec/controllers/categories_controller_spec.rb:
```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/devise'
+require 'faker'
```
these should be in rails or spec helpers

ziaulrehman40 commented on spec/controllers/categories_controller_spec.rb:
```diff
@@ -0,0 +1,167 @@
+  end
+
+  describe 'GET #index' do
+    login_admin
```
should be in before

Adeel-Tahir committed:
*remove not used files
*move require files to spec_helper file
*move login_user and login_admin in before block

Adeel-Tahir committed:
*fixed some changes
